test-services/test-probe/src/config.rs

```diff
@@ -0,0 +1,34 @@
+use crate::error::{Error,
+                   Result};
+use clap::ArgMatches;
+use std::{fs,
+          path::PathBuf};
+use toml;
+
+pub const DEFAULT_JSON_SOURCE: &str = "/hab/svc/test-probe/config/render_context_file.json";
+pub const DEFAULT_HOST: &str = "0.0.0.0";
+pub const DEFAULT_PORT: u16 = 8000;
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct Config {
```

Comment:
If you wanted an excuse to play with `structopt` it would simplify some of this boilerplate, but certainly not necessary.